test/index.js

```diff
@@ -539,6 +539,36 @@ describe('env-var', function () {
     })
   })
 
+  describe('#asSet', function () {
+    it('should return an empty set when not set', function () {
```

Comment:
Can you add a test for the failure case where a set doesn't contain unique values? I assume that's why someone would choose this over `asArray`, right?

Reply:
Great idea,
I added a test for unique values.
It does not fail however, the default behavior for Set is to just remove duplicates, I think it makes sense.
I use env vars for uuids sometimes and it would be nice to get a set from the env and use the `has` function

Reply:
Gotcha, I think that's fair if it's the default behaviour. Thanks! 👍